cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * ExternalSite
+ * @class
+ *
+ * @classdesc Initializes a new ExternalSite instance.
+ *
+ * @param {HTMLElement} element DOM Element.
+ */
+function ExternalSite( element ) {
+
+  var BASE_CLASS = 'external-site_container';
```

Comment:
Why make `BASE_CLASS` a constant? What rationale are you using in determining what is a constant?